### ISSUE TITLE
Include license information for each project

### ISSUE THREAD

**Issue by rgov** [Overconstrained/Overconstrained#7]
Selecting a library may have as much to do with its language as its license. This information should be surfaced in the overview of each project.

**Comment by hfossli** [comment]
Great idea!
on. 17. feb. 2016 kl. 14:42 skrev Ryan Govostes notifications@github.com:

**Comment by hfossli** [comment]
Good idea. I'm on it. Sent a few pull requests. Will send a couple more. 

Here's some description for what to type: https://github.com/Overconstrained/Overconstrained/commit/fa7e36959ef1fbacf524558094927c2a16595a08

**Comment by hfossli** [comment]
Whoops. This was totally unrelated. Sorry for the confusion.
